addons/arrays/fnc_selectRandom.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_selectRandom
+
+Description:
+    Select a specified amount of elements from an array without picking the same element multiple times.
+
+Parameters:
+    _array - Input Array <ARRAY>
+    _amount - Block <NUMBER>
+
+Returns:
+    New array with the specified amount of randomly selected elements <ARRAY>
+
+Example:
+    (begin example)
+        _result = [[1, 2, 3, 4, 5], 2] call CBA_fnc_selectRandom;
+        // _result => [2, 4] (random)
+    (end)
+
+Author:
+    NeilZar
+---------------------------------------------------------------------------- */
+SCRIPT(selectRandom);
+
+params [["_array", [], [[]]], ["_amount", 0, [0]]];
+
+if (_amount >= count _array) exitWith { _array };
```

Comment:
should just just be `>`?
I feel like `[[1, 2, 3, 4], 4] call CBA_fnc_selectRandom;` should still do a shuffle

Reply:
In my opion, even with `_amount` > `count _array`, this should do a shuffle, ich which case the functions return value should have less elements than `_amount`.